test/functional/kernel/nested-loop-reducesum/CMakeLists.txt

```diff
@@ -25,14 +25,18 @@ endif()
 foreach( NESTED_LOOP_BACKEND ${KERNEL_BACKENDS} )
   foreach( RESOURCE ${USE_RESOURCE} )
     foreach( NESTED_LOOP_TYPE ${NESTED_LOOPTYPES} )
-      configure_file( test-kernel-nested-loop.cpp.in
-          test-kernel${RESOURCE}nested-loop-${NESTED_LOOP_TYPE}-${NESTED_LOOP_BACKEND}.cpp )
+      if((${NESTED_LOOP_BACKEND} STREQUAL "OpenMP" OR ${NESTED_LOOP_BACKEND} STREQUAL "OpenMPTarget" OR ${NESTED_LOOP_BACKEND} STREQUAL "TBB" ) AND ${NESTED_LOOP_TYPE} STREQUAL "MultiLambdaParamReduceSum")
```

Comment:
Could this be simplified to check if the backend is in a list, then it would also be more easily extensible.

Reply:
The backend list is already comprehensive, and this logic simply filters that list for this specific test. The other reduce sum test in this directory uses the more comprehensive list, so I think we should stick with this logic for now.

Reply:
@MrBurmark Did as you suggested anyway!